CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to SUI Components
+
+The following is a set of guidelines for contributing with developping new components.
+
+## Getting started
+
+### Environment
+
+To develop new packages, you only need to install Node with NPM. Fin below the default versions this repo was built with:
+* Node: `v6+`
+* NPM: `v3`
+
+### Main commands
+
+This repo is managed with [sui-studio](https://www.npmjs.com/package/@s-ui/studio) tool.
+
+Some base commands (based on sui-studio) you should acknowledge :
+* `npm run generate <category> <name>` to create a new component.
+* `npm run co` to commit your changes with a wizard.
+* `npm run release` publish all your changes for al packages (once your PR is merge).
+
+## Naming Conventions
+
+### Event Handlers
+
+Many presentational components give feedback from user interaction calling provided callbacks.
+
+#### `on{Event}`
+
+```html
+<MyButton
+  onClick={onClickHandler}
+  onHover={onHoverHandler}
+/>
+<MyTextArea
+  onScroll={onScrollHandler}
+  onChange={onChangeHandler}
+/>
+```
+
+#### `on{SubContext}{Event}`
+For compound component, we may need to provide several callbacks.
+
+```html
+<MyDialogModal
+  onClick={onClickHandler}
```

Comment:
If we do that, then we're assuming that the `onClick` for which handler exactly? For the submit? I don't know if maybe, if it's a component with several, we should name all of them.

Reply:
I don't have a strong opinion on that. To me, it's "no subcontext for default behavior". Let's see what the rest of the guys think.

Reply:
But `any change` is even harder to determine, isn't? Because the onFilterChange is not going to overwrite the `onChange`.